src/storm/solver/IterativeMinMaxLinearEquationSolver.h

```diff
@@ -39,9 +40,9 @@ class IterativeMinMaxLinearEquationSolver : public StandardMinMaxLinearEquationS
    private:
     MinMaxMethod getMethod(Environment const& env, bool isExactMode) const;
 
-    bool solveInducedEquationSystem(Environment const& env, std::unique_ptr<LinearEquationSolver<ValueType>>& linearEquationSolver,
+    bool solveInducedEquationSystem(Environment const& env, std::unique_ptr<LinearEquationSolver<SolutionType>>& linearEquationSolver,
                                     std::vector<uint64_t> const& scheduler, std::vector<SolutionType>& x, std::vector<ValueType>& subB,
-                                    std::vector<ValueType> const& originalB) const;
+                                    std::vector<ValueType> const& originalB, OptimizationDirection dir) const;
```

Comment:
do we write this as const& ? it does not really matter for the compiler

Reply:
like this, it's consistent with the rest of the methods